### DISCUSSION TITLE
How to change color of this component?

### DISCUSSION THREAD

**Discussion by suzhiking** [michaelbrusegard/tabline.wez#58] [Q&A] [answered]
<img width="220" alt="Screenshot 2024-12-26 at 18 47 38" src="https://github.com/user-attachments/assets/f3e27089-ce33-4130-85a1-fc7cf8417dd2" />

**Comment by michaelbrusegard** [comment, marked as answer]
The separators follow the colors defined in your color scheme. To directly change them you have to either customise the theme or use another built-in theme.
